app/controllers/heritages_controller.rb

```diff
@@ -84,58 +84,58 @@ def update_service_scale
 
   def permitted_params
     params.permit([
-      :id,
-      :version,
-      :name,
-      :image_name,
-      :image_tag,
-      :before_deploy,
-      services: [
-        :name,
-        :cpu,
-        :memory,
-        :command,
-        :reverse_proxy_image,
-        :public,
-        :service_type,
-        :force_ssl,
-        :web_container_port,
-        {
-          port_mappings: [
-            :lb_port,
-            :host_port,
-            :container_port,
-            :protocol,
-            :enable_proxy_protocol
-          ],
-          hosts: [
-            :hostname,
-            :ssl_cert_path,
-            :ssl_key_path
-          ],
-          listeners: [
-            :endpoint,
-            :health_check_interval,
-            :health_check_path,
-            :rule_priority,
-            rule_conditions: [
-              :type,
-              :value
-            ]
-          ]
-        }
-      ],
-      scheduled_tasks: [
-        :schedule,
-        :command
-      ],
-      environment: [
-        :name,
-        :value,
-        :value_from,
-        :ssm_path
-      ]
-    ]).tap do |whitelisted|
+                    :id,
+                    :version,
```

Comment:
This is insane :laughing:. Should we remove the rule that made this change? Maybe it's no big deal.

Reply:
I am not sure how I can disable this 😄 
maybe it is ok but let me know what you think!

Reply:
I think you can just move this to a `PERMITTED_PARAMS` constant to avoid the nesting, because some rubocop rules combine to become really funky like this